### ISSUE TITLE
Print extra line at the end of routes inspection

### ISSUE THREAD

**Issue by jodosha** [lotus/router#82]
When multiple applications are present in a Lotus project, the output of `lotus routes` is cluttered because the lack of extra blank line between groups of routes:

Steps to reproduce:

```shell
➜ lotus new bookshelf && cd bookshelf && bundle
➜ bundle exec lotus generate app admin
➜ bundle exec lotus generate action web books#index --url=/books
➜ bundle exec lotus generate action admin users#index --url=/users
➜ bundle exec lotus routes
                Name Method     Path                           Action

                     GET, HEAD  /admin/users                   Admin::Controllers::Users::Index
                Name Method     Path                           Action

                     GET, HEAD  /books                         Web::Controllers::Books::Index
```

Expected would be:

```shell
➜ bundle exec lotus routes
                Name Method     Path                           Action
                     GET, HEAD  /admin/users                   Admin::Controllers::Users::Index

                Name Method     Path                           Action
                     GET, HEAD  /books                         Web::Controllers::Books::Index
```

---
Ref #78
/cc @davydovanton

**Comment by davydovanton** [comment]
good idea I think :+1: 
but may be is better to create smth like this:
```
                Name Method     Path                           Action
         --------------------------------------------------------------------------------------
                     GET, HEAD  /admin/users                   Admin::Controllers::Users::Index

                Name Method     Path                           Action
         --------------------------------------------------------------------------------------
                     GET, HEAD  /books                         Web::Controllers::Books::Index
```

**Comment by jodosha** [comment]
@davydovanton With that lines they look grouped like illustrated by the following screenshot:

![screen shot 2016-01-12 at 13 51 14](https://cloud.githubusercontent.com/assets/5089/12263991/ca284530-b933-11e5-8fc7-78586f5ee9ec.png)

**Comment by davydovanton** [comment]
so, @jodosha may be is better create smth like this?
```
                Name Method     Path                           Action

                     GET, HEAD  /admin/users                   Admin::Controllers::Users::Index
                     GET, HEAD  /admin/users                   Admin::Controllers::Users::Index

         --------------------------------------------------------------------------------------

                Name Method     Path                           Action

                     GET, HEAD  /books                         Web::Controllers::Books::Index
                     GET, HEAD  /books                         Web::Controllers::Books::Index
                     GET, HEAD  /books                         Web::Controllers::Books::Index
```

**Comment by jodosha** [comment]
@davydovanton It's way better. It will be probably appear like this instead:

```
                Name Method     Path                           Action

                     GET, HEAD  /admin/users                   Admin::Controllers::Users::Index
                     GET, HEAD  /admin/users                   Admin::Controllers::Users::Index

         --------------------------------------------------------------------------------------

                Name Method     Path                           Action

                     GET, HEAD  /books                         Web::Controllers::Books::Index
                     GET, HEAD  /books                         Web::Controllers::Books::Index
                     GET, HEAD  /books                         Web::Controllers::Books::Index

         --------------------------------------------------------------------------------------
```

The last router in a full stack project doesn't know that it's the last one to be inspected, so the last line will probably be there. But I don't mind.

**Comment by davydovanton** [comment]
nice, I'm pleased that we have agreed :)

I'll do it today.

**Comment by AlfonsoUceda** [comment]
@davydovanton thanks ;)

**Comment by jodosha** [comment]
Fixed by #83
